third_party/icu.BUILD

```diff
@@ -172,6 +172,56 @@ cc_binary(
     ],
 )
 
+genrule(
+    name = "pkgdata_inc",
+    outs = ["pkgdata.inc"],
+    cmd = select({
+        "@platforms//os:macos": r"""
+echo "
+GENCCODE_ASSEMBLY_TYPE=-a gcc-darwin
+SO=dylib
+SOBJ=dylib
+A=a
+LIBPREFIX=lib
+LIB_EXT_ORDER=.73.2.dylib
+COMPILE=clang -DU_ATTRIBUTE_DEPRECATED=   -DU_HAVE_STRTOD_L=1 -DU_HAVE_XLOCALE_H=1 -DU_HAVE_STRING_VIEW=1  -O2 -std=c11 -Wall -pedantic -Wshadow -Wpointer-arith -Wmissing-prototypes -Wwrite-strings  -Qunused-arguments -Wno-parentheses-equality -fno-common -c
+LIBFLAGS=-I/opt/homebrew/Cellar/icu4c/73.2/include  -dynamic
```

Comment:
What's this used for? Should it point into the Bazel build tree? Maybe add a comment about what's going on with this pkgdata.inc thing?